README.md

```
@@ -35,6 +35,7 @@ Lambdapdk is a collection of open-source Process Design Kits (PDKs) that enable
 
 | PDK | Node | Libraries | Source |
 |-----|------|-----------|--------|
+| [GT2N](lambdapdk/gt2n/README.md) | 2nm GAAFET | Standard cells (HVT/SVT/LVT/ULVT/ELVT) | Georgia Institute of Technology
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)